src/mlagility/cli/cli.py

```diff
@@ -398,8 +376,7 @@ def main():
     # we alter argv to insert the command for them.
 
     if len(sys.argv) > 1:
-        script_name, _ = decode_script_name(sys.argv[1])
-        if sys.argv[1] not in subparsers.choices.keys() and script_name.endswith(".py"):
+        if sys.argv[1] not in subparsers.choices.keys():
```

Comment:
This seems like quite a dangerous change to me. There's a lot of things the user could be doing here that aren't intended to launch a benchmark, no?

Reply:
`script_api.py` has a protection for every input script

```
        if not script.endswith(".py"):
            raise exceptions.GroqitArgError(f"Script must end with .py (got {script})")
```